pyproject.toml

```diff
@@ -42,6 +42,7 @@ torch = "^2.2.1"
 opencv-python = ">=4.9.0"
 diffusers = "^0.27.2"
 torchvision = ">=0.17.1"
+# torchvision = { path = "../../build/vision" }
```

Comment:
Should this be here? And if so, should it have a comment explaining what it's for?

Reply:
Whoops. Fixed https://github.com/huggingface/lerobot/pull/220/commits/c345056098b7aad32b4899b7a1e0df86ae4ff15b